### PR TITLE
Nikita / Phase 2 Implementing Design - Home Page 

### PR DESCRIPTION
**Pull request by nikita-deriv**
Change homepage view according to new design:  https://redmine.deriv.cloud/issues/38645#Phase_2_Card_for_Design_-_Home_Page

### CODE REVIEW

File: css/main.scss [hunk 1]
```diff
@@ -796,6 +850,109 @@ ol > p {
...
+  box-sizing: border-box;
+  content: "“";
+  position: absolute;
+  font-size: 17rem;
+  font-weight: bold;
+  z-index: -1;
+  left: 0rem;
+  top: -6.9rem;
+  color: #f2f3f4;
+}
```
Comment (prince-deriv) [inline css/main.scss:942]: @nikita-deriv  LGTM. just a small thing, can we nest this `blockquote`

```suggestion
blockquote {
  position: relative;
  text-align: left;
  
    &::after {
    box-sizing: border-box;
    content: "“";
    position: absolute;
    font-size: 17rem;
    font-weight: bold;
    z-index: -1;
    left: 0rem;
    top: -6.9rem;
    color: #f2f3f4;
  }
}

```

Comment (nikita-deriv) [reply css/main.scss:958]: Thanks, fixed with your suggestion

File: img/arrow_right.svg [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
```
Comment (reza-fs) [inline img/arrow_right.svg:1]: (throughout) let's optimize using `svgo`

**Review by prince-deriv — APPROVED** [review]

**Review by reza-fs — APPROVED** [review]

**Review by prince-deriv — APPROVED** [review]

**Review by prince-deriv — APPROVED** [review]
LGTM

**Review by mustofa-binary — APPROVED** [review]

**Pull request merged**